src/browser/service/routing.rs

```diff
@@ -79,8 +77,9 @@ pub fn url_request_handler(
     }
 }
 
-// Set up a listener that intercepts clicks on elements containing an Href attribute,
-// so we can prevent page refresh for internal links, and route internally.  Run this on load.
+/// Set up a listener that intercepts clicks on elements containing an Href
+/// attribute, so we can prevent page refresh for internal links, and route
+/// internally. Run this on load.
```

Comment:
I think there were only regular comments instead of doc comments for `setup_link_listener` and maybe `push_route` or other functions because they aren't useful for Seed users at all and should be probably marked by `pub(crate)` instead of `pub`.
Your choice how to resolve it / leave it as is.

Reply:
I'll `pub(crate)` them.